libs/blocks/global-navigation/global-navigation.js

```diff
@@ -950,6 +950,10 @@ class Gnav {
   };
 
   decorateBreadcrumbs = async () => {
+    if (getMetadata('dynamic-nav') === 'on' && window.sessionStorage.getItem('gnavSource') !== null) {
+      document.querySelector('header').classList.remove('has-breadcrumbs');
```

Comment:
You would still have some breadcrumb elements in the DOM, right? https://github.com/adobecom/milo/blob/stage/libs/utils/utils.js#L683 Should those be removed as well? Also, wouldn't initially adding the class in utils and then removing it in the later loaded gnav have an effect on CLS, since the `has-breadcrumbs` class is defined in styles.css? https://github.com/adobecom/milo/blob/stage/libs/styles/styles.css#L714

Reply:
TBH, I was wondering where in the chain of things to put this, I think you are right about the CLS, I'll look into it. 

Thank you for the good catch @overmyheadandbody

Reply:
I made the change in decorate header, and removed the code from global nav. If `has-breadcrumbs` is never added then https://github.com/adobecom/milo/blob/stage/libs/blocks/global-navigation/global-navigation.js#L953 will return null as well. 

Does this seem a better solution @overmyheadandbody? Thank you again for your input.